enterprise_subsidy/apps/content_metadata/api.py

```diff
@@ -124,29 +124,24 @@ def get_geag_variant_id_for_content(self, content_identifier, content_data):
                 variant_id = additional_metadata.get('variant_id')
         return variant_id
 
-    def enroll_by_date_for_content(self, content_data, content_mode):
+    def enroll_by_date_for_content(self, course_run_data, content_mode):
         """
-        Determines the enrollment deadline for the given ``content_data``,
-        which could be either a course or course run, hence the branching
-        on content type below.
+        Determines the enrollment deadline for the given ``course_run_data``.
 
         Args:
-            content_data: A dictionary of metadata pertaining to a course or course run.
-            content_mode: The (already computed) course mode for the content.
+            course_run_data: A dictionary of metadata pertaining to a course run.
+            content_mode: The (already computed) course mode for the course run.
 
         Returns:
             A datetime string, or possibly null.
         """
-        if content_data.get('content_type') == 'course':
-            return content_data.get('normalized_metadata', {}).get('enroll_by_date')
-
```

Comment:
I'm having trouble understanding how this changed any behavior if we only ever sent runs to this function, because in that case we'd never have run this conditional block, right?